vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -173,6 +168,10 @@ def save_guest_devices_inventory(hardware, hashes)
         # Save the hardware to force an id if not found
         hardware.save! if hardware.id.nil?
         h[:network][:hardware_id] = hardware.id
+      else
+        # ensure save_network_inventory gets called
+        # it will be deleted
+        h[:network] = {}
```

Comment:
well, this is the desired behavior I read from [save_network_inventory.rb#save_network_inventory](https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/models/ems_refresh/save_inventory.rb#L197-L204) is different.